metricbeat/etc/beat.short.yml

```diff
@@ -1,58 +0,0 @@
-########################## Metricbeat Configuration ###########################
```

Comment:
Yeah, this is on purpose, but I did want to ask you about it. I removed this file. Where is this file used?